samples/boards/nrf/mesh/onoff_level_lighting_vnd_app/prj_smp_svr.conf

```diff
@@ -9,10 +9,10 @@ CONFIG_BT_L2CAP_TX_MTU=252
 CONFIG_BT_BUF_ACL_RX_SIZE=256
 
 # Enable the Bluetooth (unauthenticated) and shell mcumgr transports.
-CONFIG_MCUMGR_SMP_BT=y
-CONFIG_MCUMGR_SMP_BT_AUTHEN=n
-#CONFIG_MCUMGR_SMP_SHELL=y
-#CONFIG_MCUMGR_SMP_UART=y
+CONFIG_MCUMGR_TRANSPORT_BT=y
```

Comment:
IMO this commit should be squashed into 1st commit, so that this patch series is bisectable.

Reply:
All samples and tests will be squashed with base/rename PR.